gtsfm/loader/astronet_loader.py

```diff
@@ -78,7 +79,7 @@ def __init__(
         if not Path(data_dir).exists():
             raise FileNotFoundError("No data found at %s." % data_dir)
         cameras, images, points3d = colmap_io.read_model(path=data_dir, ext=".bin")
-        self._calibrations, self._wTi_list, img_fnames, self._sfmtracks = self.colmap2gtsfm(
+        self._calibrations, self._wTi_list, img_fnames, self._sfmtracks, cameras_gtsfm = self.colmap2gtsfm(
```

Comment:
seeing wTi and cameras together is always a red flag ..

Reply:
Can you elaborate?

Reply:
cameras contain poses, camera.pose(), so wTi is not necessary

Reply:
Same could be said for the calibrations.

Reply:
true, so why are we returning everything here?

Reply:
Sorry, I forgot to push a commit that removed `cameras_gtsfm`; will do that.

What I meant was that even in the SceneOptimizer we are passing in calibrations and cameras. However, this is hard to avoid since it is not possible to set either the Pose or Calibration to None in `PinholeCameraCal3Bundler`. 

So, the two options are (1) pass around Calibration(s) and Pose(s) separately and construct cameras as needed, or (2) pass around Calibration(s) and Camera(s) which have redundant information.

Reply:
removed `cameras_gtsfm`